name_data/labeled/company_labeled.xml

```diff
@@ -1421,4 +1421,57 @@
   <Name><CorporationName>LeadIQ</CorporationName> <CorporationNameBranchIdentifier>Singapore</CorporationNameBranchIdentifier> <CorporationLegalType>Pte.</CorporationLegalType> <CorporationLegalType>Ltd.</CorporationLegalType></Name>
   <Name><CorporationName>Spacemob</CorporationName> <CorporationLegalType>Pte.</CorporationLegalType> <CorporationLegalType>Ltd.</CorporationLegalType></Name>
   <Name><CorporationName>NTUC</CorporationName> <CorporationName>LearningHub</CorporationName> <CorporationLegalType>Pte</CorporationLegalType> <CorporationLegalType>Ltd</CorporationLegalType></Name>
+  <Name><CorporationName>Bethlehem</CorporationName> <CorporationNameOrganization>Lutheran</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Salina</CorporationName> <CorporationName>Presbyterian</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Ebenezer</CorporationName> <CorporationNameOrganization>Lutheran</CorporationNameOrganization> <CorporationNameOrganization>Chapel</CorporationNameOrganization></Name>
+  <Name><CorporationName>Brayton</CorporationName> <CorporationNameOrganization>Methodist</CorporationNameOrganization> <CorporationName>Episcopal</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>St.</CorporationName> <CorporationName>Frances</CorporationName> <CorporationNameOrganization>Methodist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Spring</CorporationName> <CorporationName>Creek</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Abyssinian</CorporationName> <CorporationNameOrganization>Baptist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Urban</CorporationName> <CorporationName>Grace</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Temple</CorporationName> <CorporationName>B'Nai</CorporationName> <CorporationName>Abraham</CorporationName></Name>
+  <Name><CorporationName>Madison</CorporationName> <CorporationName>Avenue</CorporationName> <CorporationNameOrganization>Baptist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>St.</CorporationName> <CorporationName>Joseph</CorporationName> <CorporationNameOrganization>Basilica</CorporationNameOrganization></Name>
+  <Name><CorporationName>Trinity</CorporationName> <CorporationName>Evangelical</CorporationName> <CorporationNameOrganization>Lutheran</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
```

Comment:
`Evangelical` is likely a `CorporationNameOrganization` here, no? Seems like it's a modifier for `Lutheran`, as opposed to `Trinity`.

Reply:
Yes, I agree